docs/CONFIGID.md

```diff
@@ -0,0 +1,19 @@
+# mLRS Documentation: Configuration ID #
+
+([back to main page](../README.md))
+
+mLRS allows for 10 different configurations to be stored on the Tx module.  This is useful for when you want to use a single Tx module with multiple receivers and want to use different settings for each receiver.  For example, you may prefer to use 50 Hz mode on your copter but prefer to use 19 Hz mode on your rover. The active configuration ID is set based on the receiver number that is assigned on the MODEL SETUP page within the 'External RF' page/section of EdgeTx/OpenTx. This option is only available when using CRSF or mBridge protocols, SBUS will only use configuration ID 0.
+
+## Changing the Receiver Number
+
+Within EdgeTx/OpenTx, navigate to the MDL->MODEL SETUP page and then select the 'External RF' Section:
+
+<img src="images/configIDModelSetup.bmp" width="360px">
```

Comment:
name does not match file name

Reply:
Good catch - seems the image lookup is case insensitive, the preview was able to load the image fine.  But have fixed the reference to be consistent.